### ISSUE TITLE
Errors when computing

### ISSUE THREAD

**Issue by DrDiban** [AntonMu/TrainYourOwnYOLO#76]
Hi Anton,

I still a beginner when it comes to programming. I have read through what you have wrote and confidence that I have followed word for word your instructions. I having trouble when running the train_yolo file. The training starts with bunch of errors and before epoch 1/51 finish computing it stops.
I need your help to figure up where it is going wrong.

These are the major errors/warning that I suspect of  might be causing the error:
1) There are bunch of warnings that shows the deprecated tf functions(dont think this is the problem)

2) Some weight are not loaded since there are not matching in dimensions 
C:\Users\Dr Diban\TrainYourOwnYOLO\env\lib\site-packages\keras\engine\saving.py:1140: UserWarning: Skipping loading of weights for layer conv2d_59 due to mismatch in shape ((1, 1, 1024, 18) vs (255, 1024, 1, 1)).
  weight_values[i].shape))
C:\Users\Dr Diban\TrainYourOwnYOLO\env\lib\site-packages\keras\engine\saving.py:1140: UserWarning: Skipping loading of weights for layer conv2d_59 due to mismatch in shape ((18,) vs (255,)).
  weight_values[i].shape))
C:\Users\Dr Diban\TrainYourOwnYOLO\env\lib\site-packages\keras\engine\saving.py:1140: UserWarning: Skipping loading of weights for layer conv2d_67 due to mismatch in shape ((1, 1, 512, 18) vs (255, 512, 1, 1)).
  weight_values[i].shape))
C:\Users\Dr Diban\TrainYourOwnYOLO\env\lib\site-packages\keras\engine\saving.py:1140: UserWarning: Skipping loading of weights for layer conv2d_67 due to mismatch in shape ((18,) vs (255,)).
  weight_values[i].shape))
C:\Users\Dr Diban\TrainYourOwnYOLO\env\lib\site-packages\keras\engine\saving.py:1140: UserWarning: Skipping loading of weights for layer conv2d_75 due to mismatch in shape ((1, 1, 256, 18) vs (255, 256, 1, 1)).
  weight_values[i].shape))
C:\Users\Dr Diban\TrainYourOwnYOLO\env\lib\site-packages\keras\engine\saving.py:1140: UserWarning: Skipping loading of weights for layer conv2d_75 due to mismatch in shape ((18,) vs (255,)).
  weight_values[i].shape)) 

3) Pillow error-I also used your cat pictures for training but still get the below errors
C:\Users\Dr Diban\TrainYourOwnYOLO\env\lib\site-packages\PIL\Image.py:989: UserWarning: Palette images with Transparency expressed in bytes should be converted to RGBA images

4) Remapper error-E tensorflow/core/grappler/optimizers/meta_optimizer.cc:533] remapper failed: Invalid argument: Subshape must have computed start >= end since stride is negative, but is 0 and 2 (computed from start 0 and end 9223372036854775807 over shape with rank 2 and stride-1)

5)Finally the memory error- W tensorflow/core/framework/cpu_allocator_impl.cc:81] Allocation of 177209344 exceeds 10% of system memory.

Hope that you can shed some light.

**Comment by AntonMu** [comment]
Hi @DrDiban - thanks for your message. 

Two things:
 * Don't use spaces in your folder name. Just put everything under `C:\TrainYourOwnYOLO\...` to begin with.
 * If you would like to have help, I recommend using the [issue template](https://github.com/AntonMu/TrainYourOwnYOLO#filing-an-issue).

**Comment by DrDiban** [comment]
Hi Anton,

Thanks for your reply. I have carried out the first step as per your reply but I still get the same error. I will proceed to fill up the issue template as suggested.

**Comment by DrDiban** [comment]
Hi Anton,

A quick update, for error 3, when I used the cat pictures used in the existing folder it does not appear any longer. Yesterday when I say that I was still  getting an error using your pictures was due to my error, I was exporting the files in json format. 

Also I managed to get the model running with small sample size (with other warnings apart from item 3 still remains). So I guessing my computer low spec  causing the model to crash when I am using 100 pictures.

I have a question regarding the AWS that you suggested, is it a free service or a paid service? Cause when I registered I did not get any vCPU capacity.
